sites/footers/reports/a11y-1.json

```
@@ -0,0 +1,905 @@
+        "WCAG21Tech:general/G186",
+        "WCAG21Tech:general/G191",
+        "WCAG21Tech:failures/F16",
+        "WCAG21Tech:failures/F47",
+        "WCAG21Tech:failures/F4",
+        "WCAG21Tech:failures/F50",
+        "WCAG21Tech:failures/F7"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.3.1 Three Flashes or Below Threshold (Level A)",
+      "earl:test": "WCAG20:three-flashes-or-below-threshold",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G19",
+        "WCAG21Tech:general/G176",
+        "WCAG21Tech:general/G15"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.1 Bypass Blocks (Level A)",
+      "earl:test": "WCAG20:bypass-blocks",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G1",
+        "WCAG21Tech:general/G123",
+        "WCAG21Tech:general/G124"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.2 Page Titled (Level A)",
+      "earl:test": "WCAG20:pause-stop-hide",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": ["WCAG21Tech:general/G88", "WCAG21Tech:failures/F25"],
+      "earl:mode": "earl:semiAuto",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.3 Focus Order (Level A)",
+      "earl:test": "WCAG20:focus-order",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G59",
+        "WCAG21Tech:failures/F44",
+        "WCAG21Tech:failures/F85"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.4 Link Purpose (In Context) (Level A)",
+      "earl:test": "WCAG20:link-purpose-in-context",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G91",
+        "WCAG21Tech:general/G189",
+        "WCAG21Tech:general/G53",
+        "WCAG21Tech:failures/F63",
+        "WCAG21Tech:failures/F89"
+      ],
+      "earl:mode": "earl:semiAuto",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 2.4.5 Multiple Ways (Level AA)",
+      "earl:test": "WCAG20:multiple-ways",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G125",
+        "WCAG21Tech:general/G64",
+        "WCAG21Tech:general/G63",
+        "WCAG21Tech:general/G161",
+        "WCAG21Tech:general/G126",
+        "WCAG21Tech:general/G185"
+      ],
```

According to the recently updated instruction, in this case there should be no technique listed.

```suggestion
      "techniques": [ ],
```

---

Those technique outcome was marked "can't tell"